docs/reference/commandline/stack_rm.md

```diff
@@ -7,6 +7,12 @@ Remove one or more stacks
 
 `docker stack rm`, `docker stack remove`, `docker stack down`
 
+### Options
+
+| Name             | Type | Default | Description                   |
+|:-----------------|:-----|:--------|:------------------------------|
+| `-d`, `--detach` |      |         | Do not wait for stack removal |
```

Comment:
Interesting; for some reason it doesn't show the default here 🤔 


`--help` shows the correct output, but the markdown generator doesn't 🤔 Any ideas @crazy-max ?

```bash
./build/docker stack rm --help

Usage:  docker stack rm [OPTIONS] STACK [STACK...]

Remove one or more stacks

Aliases:
  docker stack rm, docker stack remove, docker stack down

Options:
  -d, --detach   Do not wait for stack removal (default true)
```

Reply:
I think it makes sense to not show default as this is a bool flag anyway? (like `--pull` or `--push`)

Reply:
Oh my bad this is true by default so opt-out flag right?

Reply:
@crazy-max Yes, it's true by default and the user can use `--detach=false` if they want to wait for stack removal to complete

Reply:
It's the same for `service create/update` where we already have implemented this flag in https://github.com/moby/moby/pull/31144

https://github.com/docker/cli/blob/79fa65e7b58b081dfc2316a1a3c0f602f4d4c01a/docs/reference/commandline/service_create.md?plain=1#L6-L16

Reply:
Crazy thought, how about "--wait"?

Reply:
So the intent here is to (like `docker service create`) flip the default at some point, so make `stack deploy` act the same as `docker compose up`.

I don't think the markdown here is a real blocker for the PR (we can fix in "post"), but I noticed it, and was curious what went wrong here (looks like a bug in the markdown generator potentially?)

Reply:
I agree, we can investigate further in a separate issue, as it seems like a generalized bug in the markdown generator.

Is there anything else left to be done in this PR?